src/modules/did.js

```diff
@@ -91,6 +106,16 @@ class DIDModule {
     return `${DockDIDQualifier}${did}`;
   }
 
+  /**
+   * Fetches the DIDs attestations IRI from the chain
+   * @param {string} hexId - DID in hex format
+   * @return {Promise<object>} The DID attestations
+   */
+  async getAttests(hexId) {
+    const attests = await this.api.query.attest.attestations(hexId);
+    return attests.iri.isSome ? u8aToString(hexToU8a(attests.iri.toString())) : '';
```

Comment:
fyi, the bytes on-chain aren't guaranteed to be valid utf8.

Reply:
are they guaranteed to always be strings (human readable or?) or could they be "random" bytes? if non-strings are expected, then i can remove the u8aToString conversion and just return the u8a from hex instead, moving that string check to the test. wdyt?

Reply:
They are only meaningful as strings. If they are "random" bytes they are invalid. It is possible to post an invalid string on-chain because the chain doesn't care.

It does make sense to always attempt a conversion to string. Also makes sense to be conscious of the behavior of `u8aToString`. Maybe it throws an error? Maybe it returns a string no matter what (that could be bad?)? I don't know.

Reply:
It'll return a string no matter what given that a string is just a uint8 array anyway behind the scenes, it essentially will convert the Uint8Array type to String.